src/config/assistant.ts

```diff
@@ -45,7 +45,7 @@ export const ASSISTANT_CONFIG: AssistantConfig = [
       {
         title: 'Inflation',
         description: [
-          'DOT is inflationary; there is no maximum number of DOT.',
+          'CERE is inflationary; there is no maximum number of CERE.',
```

Comment:
Is it true?

Reply:
Yes, it's.
For example, current inflation is 1.1885% You could check it in https://explorer.cere.network/#/staking